addon/services/i18n.js

```diff
@@ -277,36 +272,25 @@ const I18nService = Service.extend({
   },
 
   _initLibrary() {
```

Comment:
Primary change is here. Now that i18next returns a promise from `init`, there's no need to wrap it with our own promise.